src/app/components/pages/send-skycoin/send-verify/send-verify.component.ts

```diff
@@ -39,8 +42,7 @@ export class SendVerifyComponent {
 
   private onSuccess() {
     this.sendButton.setSuccess();
-    this.walletService.loadBalances();
```

Comment:
This was good here. In a previous comment I said that the balance had to be updated just after sending the transaction. This was doing precisely that, but I did not notice.